src/components/common/DetailComment.vue

```diff
@@ -16,67 +16,61 @@
   </v-card>
 </template>
 
-<script lang="ts">
-import Vue from 'vue'
-import { Component, Prop, Watch, Emit } from 'vue-property-decorator'
-import { Debounce } from 'vue-debounce-decorator'
```

Comment:
I created [13853](https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/13853) to investigate what we can do about this.

This is pretty useful to limit the amount of validation/events that need to be processed.

Reply:
Oh wait -- the lodash method works?

Reply:
Ya it looks like the lodash method works for this.